app/models/user.rb

```diff
@@ -101,87 +101,92 @@ def can_use_api?
 
   # Can this user create a new Patient?
   def can_create_patient?
-    role?(Roles::ENROLLER) || role?(Roles::PUBLIC_HEALTH_ENROLLER)
+    role?(Roles::ENROLLER) || role?(Roles::CONTACT_TRACER) || role?(Roles::PUBLIC_HEALTH_ENROLLER) || role?(Roles::SUPER_USER)
   end
 
   # Can this user view a Patient?
   def can_view_patient?
-    role?(Roles::ENROLLER) || role?(Roles::PUBLIC_HEALTH) || role?(Roles::PUBLIC_HEALTH_ENROLLER)
+    role?(Roles::ENROLLER) || role?(Roles::PUBLIC_HEALTH) || role?(Roles::CONTACT_TRACER) || role?(Roles::PUBLIC_HEALTH_ENROLLER) || role?(Roles::SUPER_USER)
```

Comment:
Not that important, but what if we created this:

```
def roles?(roles)
  roles.include?(self.role)
end
```

and then did something like:

```
def can_view_patient?
  roles?(Roles::ENROLLER, Roles::PUBLIC_HEALTH, Roles:: CONTACT_TRACER, Roles::PUBLIC_HEALTH_ENROLLER, Roles:: SUPER_USER)
end
```

Reply:
Yeah that's definitely do-able and probably better because it's fewer calls. My first thought was that it feels less readable to me just because I like seeing the explicit OR logic, but that is definitely not a strong reason really given the performance increase. @holmesie Any preferences?